controls/srg_gpos/SRG-OS-000072-GPOS-00040.yml

```diff
@@ -6,8 +6,12 @@ controls:
             number of characters when passwords are changed.
         rules:
             - accounts_password_pam_difok
+            - var_password_pam_difok=8
             - accounts_password_pam_enforce_root
             - accounts_password_pam_maxclassrepeat
+            - var_password_pam_maxclassrepeat=4
             - accounts_password_pam_maxrepeat
+            - var_password_pam_maxrepeat=3
             - accounts_password_pam_minclass
+            - var_password_pam_minclass=4
```

Comment:
You can remove these variables from this file then: https://github.com/ComplianceAsCode/content/blob/master/controls/srg_gpos.yml#L19